modules/core/02-client/types/events.go

```diff
@@ -28,6 +28,7 @@ var (
 	EventTypeUpdateClientProposal  = "update_client_proposal"
 	EventTypeUpgradeChain          = "upgrade_chain"
 	EventTypeUpgradeClientProposal = "upgrade_client_proposal"
+	EventTypeRecoverClient         = "recover_client"
```

Comment:
If we use a separate event, we need to inform relayers, so that they can handle both the `update_client_proposal` and the new `recover_client`. In this case we can mark `update_client_proposal` as deprecated.

Reply:
looking into the golang relayer codecase, i don't even see an `UpgradeClientProposal` event type being handled, only `UpgradeClient` and `UpdateClientProposal`

Reply:
Well, we could always make the change and revert if it is problematic!

Reply:
sgtm 🚀